src/index.ts

```diff
@@ -109,5 +109,15 @@ ${kleur.bold(`${tada} Capacitor plugin generated! ${tada}`)}
 
 Next steps:
   - ${kleur.cyan(`cd ${details.dir}/`)}
+  - Open ${kleur.bold('CONTRIBUTING.md')} to learn about the npm scripts
+  - Continue following these docs for plugin development: ${kleur.bold(
+    'https://capacitorjs.com/docs/plugins/workflow',
```

Comment:
```suggestion
    'https://capacitorjs.com/docs/v3/plugins/workflow',
```
Should it point to the v3 docs since the generator is for v3 compatible plugins?

Reply:
Yeah!